INSTALL.md

```diff
@@ -85,14 +90,14 @@ hrim generate models/actuator/gripper/gripper.xml
 ```
 ### Compile the workspace
 
-Please make sure you are not sourcing ROS1 workspaces via `bashrc` or any other way.
+Please make sure you are not sourcing ROS1 workspaces via `bashrc` or any other way. Also make sure you are not sourcing any provisioning script from other ROS2 distribution compliant gym-gazebo2 installation, e.g. gym-gazebo2 `crystal`.
```

Comment:
dashing

Reply:
Id say this is okay. I want to say be careful with using provisioning script form other branches, e.g. crystal. because that would try to source `/opt/ros/crystal/setup.bash` and not dashing.

Reply:
It's ok for me.